Formula/g/gnupg.rb

```diff
@@ -39,21 +51,28 @@ class Gnupg < Formula
     depends_on "gettext"
   end
 
-  # Backport fix for missing unistd.h
-  patch do
-    url "https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gnupg.git;a=patch;h=1d5cfa9b7fd22e1c46eeed5fa9fed2af6f81d34f"
-    sha256 "610d0c50004e900f1310f58255fbf559db641edf22abb86a6f0eb6c270959a5d"
-  end
-
   def install
     libusb = Formula["libusb"]
     ENV.append "CPPFLAGS", "-I#{libusb.opt_include}/libusb-#{libusb.version.major_minor}"
+    ENV.append "LDFLAGS", "-L#{libusb.opt_lib}"
+
+    readline = Formula["readline"]
+    ENV.append "CPPFLAGS", "-I#{readline.opt_include}"
+    ENV.append "LDFLAGS", "-L#{readline.opt_lib}"
+
+    maintenance_args = []
+    unless File.exist?("configure") # i.e., this is HEAD
+      system "./autogen.sh"
+      maintenance_args = ["--enable-maintainer-mode"]
+    end
```

Comment:
Let's guard this with a check for HEAD builds then, this assumption seems dangerous

Reply:
Fixed as requested